### DISCUSSION TITLE
We need bedrock 😭

### DISCUSSION THREAD

**Discussion by 4mlsx** [xnebulr/Minecraft-Default-Dark-Mode#117] [Ideas]
It will be great for bedrock player

**Comment by nebuIr** [comment]
I'm working on a version for Bedrock. It will probably be ready in a few weeks.

**Comment by aricloverEXTRA** [comment]
Hey @4mlsx here’s a workaround solution!

Dark Mode (Bedrock Resource Pack) by Offroaders123: https://mcpedl.com/dark-mode-resource-pack/
